tools/depends/target/Toolchain.cmake.in

```diff
@@ -12,7 +12,6 @@ set(PLATFORM "@target_platform@")
 if(OS STREQUAL linux)
   set(CMAKE_SYSTEM_NAME Linux)
   set(CORE_SYSTEM_NAME linux)
-  set(CORE_PLATFORM_NAME @target_platform@)
```

Comment:
I'll answer this as this was my recommendation.

Currently the only reason we have this in the toolchain is to pass what is a CI only setting. The only reason this (and the target_platform changes you noted) exist is to have jenkins not build x11 for arm/aarch64 as a CI decision. @lrusak confirmed that we dont build x11 for arm/aarch only because "cross compiling x11 is difficult", not because it isnt possible. imo such a "CI only" choice should be made at the CI level, not in the codebase.

I implemented the changes required at the jenkins job level (adaptive CORE_PLATFORM_NAME name based on CI job for all build* jobs as a var passed to parent LINUX job) that would make this change work, but that was before i stepped back with any decisions around this regarding linux/platform etc (ie just not my place to recommend stuff around linux stuff).

That was the reasoning for this recommendation, but it doesnt bother me if you do not want to go ahead with such a change.

Reply:
Drop this now. Not needed/wanted anymore

Reply:
@fuzzard This was already removed.

Reply:
Still shows as a change in the PR

Reply:
@fuzzard Done. Hopefully